metacat-functional-tests/metacat-test-cluster/docker-compose.yml

```diff
@@ -61,6 +61,14 @@ services:
           - ./datastores/cassandra/:/init
         labels:
           - "com.netflix.metacat.oss.test"
+    druid:
+        hostname: druidserver
+        image: druidio/example-cluster
+        ports:
+          - '3000:8082'
```

Comment:
Why are you hardcoding the port mapping for localhost here? The rest of them are dynamic and inside the docker-compose network you can use the default ports?